main.go

```diff
@@ -204,6 +205,11 @@ out:
 	log.Printf("Player %q got connected\n", client.Player.Nickname)
 	server.ClientLoggedIn(client.Nickname, client)
 	client.ReadLinesInto(stopCh)
+	err := termbox.Init(client)
```

Comment:
FIle etsi o client mpainei sto termbox kai den epireazei ton server. an kai to gegonos oti o server epireazete brika oti ginete apo allo simio stin Init. Prepei na tin allaksoume teleiws. Tha to checkarw simera. Na to kanoume merge auto les ?